src/language/zh-CN.json

```diff
@@ -563,7 +563,7 @@
     "label_postfix_multisig": "(多重)",
     "label_signed_by": "由联署人签署:",
     "latest_news_articles": "最新资讯",
-    "ledger_connected_other_app": "Another app is already connected to your Ledger device. Please close it then try again.",
+    "ledger_connected_other_app": "Ledger device cannot access the path, please use your ledger to verify your address or close any other opened apps then try again.",
```

Comment:
Are we not translating any more?

Reply:
we're leaving them in English for native speakers contributors to easily pick them up and add the proper translation.

Reply:
shall we google translate for now, or raise an issue for it for tracking purposes. otherwise, it will get forgotten